src/components/views/elements/AppPermission.tsx

```diff
@@ -137,20 +140,22 @@ export default class AppPermission extends React.Component<IProps, IState> {
 
         return (
             <div className="mx_AppPermission">
-                <div className="mx_AppPermission_bolder mx_AppPermission_smallText">{_t("Widget added by")}</div>
-                <div>
-                    {avatar}
-                    <h4 className="mx_AppPermission_bolder">{displayName}</h4>
-                    <div className="mx_AppPermission_smallText">{userId}</div>
-                </div>
-                <div className="mx_AppPermission_smallText">{warning}</div>
-                <div className="mx_AppPermission_smallText">
```

Comment:
Since `Heading` component takes care of the font size, `mx_AppPermission_smallText` can be removed by setting the font size `mx_AppPermission_smallText` specifies by default (see [here](https://github.com/matrix-org/matrix-react-sdk/pull/10954/files#diff-a9bc8899b6e217c49aaa8a7ec1a54862b6f6a39e09d5e9d4b5f54e8958843e4eR19)).